lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -42,19 +33,42 @@ def call(package)
 
         private
 
+        def boolean_error(manifest)
+          {
+            requirements_only: 'requirementsOnly',
```

Comment:
good point! Will try to eliminate JSON key names from everywhere except manifest.rb